requirements/bench.txt

```diff
@@ -13,3 +13,4 @@ tqdm
 torchvision
 pyomo>=5.7
 IOHexperimenter>=0.2.8
+pyomo.solvers
```

Comment:
I guess this does not exist

Reply:
it does exist but in conda-forge; removed.